lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -108,6 +110,13 @@ def instructor_dashboard_2(request, course_id):
     if course_mode_has_price and (access['finance_admin'] or access['sales_admin']):
         sections.append(_section_e_commerce(course, access, paid_modes[0], is_white_label))
 
+    # Certificates panel
+    # This is used to generate example certificates
+    # and enable self-generated certificates for a course.
+    certs_enabled = CertificateGenerationConfiguration.current().enabled
+    if certs_enabled and access['admin']:
```

Comment:
user.is_staff and GlobalStaff().has_user() are the same, right? Just wanted to make sure. And CertificateGenerationConfiguration.current() will never be None, right?